packages/storybook/stories/Adapters/Adapters.stories.mdx

```diff
@@ -12,9 +12,9 @@ const ChatAdapterExampleText = require('!!raw-loader!./snippets/ChatAdapterExamp
 
 Composites deliver turn-key communication experiences for developers.
 They make it easy for developers to add end-to-end communication experiences to their apps with only a couple lines of code.
-To power Composites, the UI library provides `Adapters`, one for the calling composite [here](./?path=/docs/composites-call-basicexample--basic-example) and one for chat composite
+To power Composites, the UI library provides `Adapters`, one for the `CallComposite` [here](./?path=/docs/composites-call-basicexample--basic-example) and one for `ChatComposite`
 [here](./?path=/docs/composites-chat-basicexample--basic-example).
-These adapters make use of the stateful clients for calling and chat in conjunction with React Hooks like usePropsFor to provide developers a single interface to configure Composites.
+These adapters make use of the stateful clients for calling and chat in conjunction with React Hooks like `usePropsFor` to provide developers a single interface to configure composites.
 
 ## Setting Up Adapters
```

Comment:
Should this be "Setting up adapters" or did we agree to capitalize all heading words. I just look at Twilio docs and their headings do not capitalize all words. 
![image](https://user-images.githubusercontent.com/79475487/145085913-840dd7b4-f737-4d12-85a4-de6ae73d3b58.png)